theano/scalar/basic.py

```diff
@@ -2166,6 +2168,14 @@ def c_code(self, node, name, (x,), (z,), sub):
         return "%(z)s = -%(x)s;" % locals()
 neg = Neg(same_out, name='neg')
 
+pprint.assign(add, printing.OperatorPrinter('+', -2, 'either'))
+pprint.assign(mul, printing.OperatorPrinter('*', -1, 'either'))
+pprint.assign(sub, printing.OperatorPrinter('-', -2, 'left'))
+pprint.assign(neg, printing.OperatorPrinter('-', 0, 'either'))
+pprint.assign(true_div, printing.OperatorPrinter('/', -1, 'left'))
+pprint.assign(int_div, printing.OperatorPrinter('//', -1, 'left'))
+pprint.assign(pow, printing.OperatorPrinter('**', 1, 'right'))
+pprint.assign(mod, printing.OperatorPrinter('%', -1, 'lef'))
```

Comment:
Oops, I missed that one. @mohammadpz Can you send another PR to fix the
typo?

On Wed, Jan 21, 2015 at 2:02 PM, Pascal Lamblin <notifications@github.com>
wrote:

> In theano/scalar/basic.py
> <https://github.com/Theano/Theano/pull/2407#discussion_r23324367>:
>
> > @@ -2166,6 +2168,14 @@ def c_code(self, node, name, (x,), (z,), sub):
> >          return "%(z)s = -%(x)s;" % locals()
> >  neg = Neg(same_out, name='neg')
> >
> > +pprint.assign(add, printing.OperatorPrinter('+', -2, 'either'))
> > +pprint.assign(mul, printing.OperatorPrinter('*', -1, 'either'))
> > +pprint.assign(sub, printing.OperatorPrinter('-', -2, 'left'))
> > +pprint.assign(neg, printing.OperatorPrinter('-', 0, 'either'))
> > +pprint.assign(true_div, printing.OperatorPrinter('/', -1, 'left'))
> > +pprint.assign(int_div, printing.OperatorPrinter('//', -1, 'left'))
> > +pprint.assign(pow, printing.OperatorPrinter('**', 1, 'right'))
> > +pprint.assign(mod, printing.OperatorPrinter('%', -1, 'lef'))
>
> Should be 'left', not 'lef', I think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2407/files#r23324367>.
>